### PR TITLE
Implementations on multiple hooks

### PR DESCRIPTION
**Pull request by kasunkavinda**
Implementations on multiple hooks

### CODE REVIEW

File: src/services/Evernode.js [hunk 1]
```diff
@@ -47,27 +47,29 @@ export const useEvernode = () => {
     return useContext(EvernodeContext)
 }
 
-const registryAddress = process.env.REACT_APP_REGISTRY_ADDRESS;
+const governorAddress = process.env.REACT_APP_REGISTRY_ADDRESS;
```
Comment (kithminisg) [inline src/services/Evernode.js:50]: REACT_APP_REGISTRY_ADDRESS => REACT_APP_GOVERNOR_ADDRESS

File: src/pages/Registry/index.js [hunk 2]
```diff
@@ -146,24 +146,71 @@ export default function Registry() {
         tableHeadings: tableHeadings,
         tableValues: rewardConfigTableValues
       });
+
```
Comment (kithminisg) [inline src/pages/Registry/index.js:149]: Can we give some additional detail in the tooltips?

File: src/pages/Registry/index.js [hunk 3]
```diff
@@ -146,24 +146,71 @@ export default function Registry() {
...
+      const hookConfigTableValues = [
+        {
+          key: 'Governor Address',
+          value: <Tooltip title="Governor address"><span>{governorAddress}</span></Tooltip>,
```
Comment (kithminisg) [inline src/pages/Registry/index.js:153]: Can we give some additional detail in the tooltips?

Comment (kasunkavinda) [reply src/pages/Registry/index.js:153]: I have now added "Hook" to the description. Can we add some more?

**Review by chalith — APPROVED** [review]

**Pull request merged**